components/data-oriented-programming/code/java-bbe/type-constraints.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Data validation at the boundary'
-description: Boundary data validation is a critical aspect of data-oriented programming as it ensures that only valid and reliable data is allowed into the system enhancing data integrity, downstream processing, and security. <br><br>Ballerina with its built-in language features handle the validation process automatically, ensuring that only valid data is accepted.
+description: Boundary data validation is a critical aspect of data-oriented programming as it ensures that only valid and reliable data is allowed into the system enhancing data integrity, downstream processing, and security. <br><br>Ballerina with its built-in language features handles the validation process automatically, ensuring that only valid data is accepted.
```

Comment:
```suggestion
description: Boundary data validation is a critical aspect of data-oriented programming as it ensures that only valid and reliable data is allowed into the system enhancing data integrity, downstream processing, and security. <br><br>Ballerina, with its built-in language features, handles the validation process automatically, ensuring that only valid data is accepted.
```

Reply:
These commas seem to be redundant. The same applies as mentioned in https://github.com/ballerina-platform/ballerina-dev-website/pull/7447#discussion_r1292677644.

Better to rewords as follows: `With its built-in language features, Ballerina handles the validation process automatically ensuring that only valid data is accepted.

Reply:
Same comment. https://github.com/ballerina-platform/ballerina-dev-website/pull/7447#discussion_r1294690780

Reply:
Reverted to the original.

Reply:
I don't think commas are redundant.